docs/docs/performance.md

```diff
@@ -9,33 +9,41 @@ This means that performance inside an enclave will normally be lower than with a
 the performance difference and how they vary between variety of benchmarks taken from the 
 [Computer Language Benchmarks Game](https://salsa.debian.org/benchmarksgame-team/benchmarksgame/).
 
+The configuration used to perform the benchmarks were:
+1) Warmup iteration: 10 
+2) Number of iteration for each benchmark: 100
+3) Azure machine series: Standard DC4s v3 (4 vcpus - Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz, 32 GiB memory)
+4) Operating system: Ubuntu 20.04.5 LTS 
+5) Conclave version: 1.4
```

Comment:
Let's change this to 1.3 and raise this PR on release/1.3 branch instead. We want to update the doscite with these figures, rather than delaying until 1.4 comes out.